src/bitcoin-cli.cpp

```diff
@@ -298,30 +296,22 @@ class GetinfoRequestHandler: public BaseRequestHandler
 class NetinfoRequestHandler : public BaseRequestHandler
 {
 private:
-    bool IsAddrIPv6(const std::string& addr) const
+    const std::vector<std::pair<std::string, uint8_t>> m_networks{{"ipv4", 0}, {"ipv6", 1}, {"onion", 2}};
```

Comment:
As these IDs are always consecutively, and you iterate over it linearly, why not simply
```
const std::vector<std::string> m_networks{"ipv4","ipv6","onion"};
```
alternatively, *lookup* would be easier with a `std::map`.

Reply:
Good idea--done, and changed the new data structures in the last commit to fixed-sized `std::array`.

Reply:
Good idea to flatten it--done, and changed the new data structures in the last commit to fixed-sized std::arrays.